### ISSUE TITLE
ffpython doesn't support setPrefs

### ISSUE THREAD

**Issue by sergey-s-betke** [fontforge/fontforge#2152]
ffpython doesn't support `setPrefs`:

```
fontforge.setPrefs ('GenerateHintWidthEqualityTolerance', 4)
fontforge.setPrefs ('StandardSlopeError', 3)
fontforge.setPrefs ('HintBoundingBoxes', 1)
fontforge.setPrefs ('HintDiagonalEnds', 1)
fontforge.setPrefs ('DetectDiagonalStems', 1)
fontforge.setPrefs ('InstructDiagonalStems', 1)
fontforge.setPrefs ('InstructSerifs', 0)
fontforge.setPrefs ('InstructBallTerminals', 0)
fontforge.setPrefs ('InterpolateStrongPoints', 0)
fontforge.setPrefs ('CounterControl', 0)
```

generate error:

```
Traceback (most recent call last):
  File "tools/generate-autohinted-ttf.py", line 17, in <module>
    fontforge.setPrefs ('GenerateHintWidthEqualityTolerance', 1)
NameError: Unknown preference item in SetPrefs: GenerateHintWidthEqualityTolerance
make: *** [release/ttf/GOST2.304-81TypeA-Regular.ttf] Ошибка 1
```

for all preferences. But if I run this script with `fontforge.exe` (not with the `ffpython.exe`) - script run successful.

How I can set preferences for current script session for `ffpython.exe`?

**Comment by sergey-s-betke** [comment]
Next ffpython issues. With 2014.12 release I had no problems. But with 2015.02 release `generateTtc` in ffpython create zero length file, but `generateTtc` in fontforge.exe create correct .ttc file.

And `generate` can`t create .woff file with ffpython, with fontforge.exe - can create.

Problem just with 2015.02 release!

**Comment by jtanx** [comment]
When running from the fontforge executable, the 'ui' functions are called, e.g. `PrefsUI_GetPrefs` over `NOUI_GetPrefs` when ffpython is used.

**Comment by jtanx** [comment]
This is not specific to Windows. Commits like 33c80adc0a2540f4ea5d4d3b59e665e8a3e updated `prefs.c` but did not update `noprefs.c` with the same preference options. Does anyone know why there are separate preference functions depending on if there's a ui or not anyway?

**Comment by skef** [comment]
This was fixed in 271b09abe8
